lib/brewdler/brew_dumper.rb

```diff
@@ -0,0 +1,42 @@
+require "json"
+require "version"
```

Comment:
What's the use-case of it below? Being stupid but it's not totally clear to me why it's required.